script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -1,32 +1,32 @@
 <Page1>
 I can't go down like this.
-The ground below is completely filled with blue flames.
+The floor of the pit is completely covered in blue flames.
 <Page2>
 A tall vampire stands at the center.
 Vlov Arkhangel.
-He stands in the middle of the crater without a single scratch.
+He was right at the center of that explosion, but there's not a scratch on him.//あの爆撃のただ中にいて、傷一つ負っていない。
 By human standards, he's practically an immortal monster.
-And then...
+And then―――
 <Page3>
 There is a white figure toying with the man from the front.
-The beautiful hunter darts through the garden of flickering flames and debris.
+A beautiful huntress darting through the garden of flickering flames and debris.
 <Page4>
-Of course, that white figure belongs to Arcueid.
+Of course, that white figure is Arcueid.
 She leaps across the river of flames and closes in on Vlov.
 <Page5>
-He has a crude and deadly weapon gripped in his hand.
-Although it has the appearance of a long sword, the blade is that of a billhook.
+He has a crude yet deadly weapon gripped in his hand.
+Although it has the appearance of a long sword, the blade is that of a machete.
 Arcueid on the other hand is fighting empty-handed.
-Using the force of her legs alone, she closes in on Vlov. And then...
+Using the force of her legs alone, she closes in on Vlov. And then―――
 <Page6>
-She exchanges blows with Vlov's billhook using just her claws, causing the tall figure to retreat...!
+She exchanges blows with Vlov's machete using just her claws, forcing the tall figure to retreat...!
 <Page7>
-A single wound can be seen on his arm, so deep that the bone within is visible.
-It is only there for an instant. The arm sliced open by Arcueid's claw is healed by the very blood that pours forth.
+A single gash can be seen on his arm, so deep that the bone within is visible.
```

Comment:
Hm, I might agree with underneath if the gash where on the skin, but since it's arm it feels more like the bone is within the arm than beneath the arm? Not a strong preference.

Reply:
I'm in agreement with 'within'

Reply:
"on his arm" describes the surface though. "underneath" the surface. I'd agree if it was literally "within his arm" or something.

Reply:
I guess I can see both - Yusuke and I are associating "A single gash ... on his **arm**, so deep ... **within**" vs "A single **gash** ... on his arm, so deep ... **beneath**". Can change it to underneath.